app/forms/course_details_form.rb

```diff
@@ -97,6 +100,10 @@ def require_age_range?
     !trainee.early_years_route?
   end
 
+  def require_study_mode?
```

Comment:
I think we can just `delegate :require_study_mode?, to: :trainee` and remove the method definition here